observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc

```diff
@@ -0,0 +1,150 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-configuring-lokistack-otlp"]
+= OTLP data ingestion in Loki
+include::_attributes/common-attributes.adoc[]
+:context: log6x-configuring-lokistack-otlp
+
+toc::[]
+
+Loki 3.0 introduces an API endpoint using the OpenTelemetry Protocol (OTLP), providing a new way to ingest log entries into Loki. This endpoint complements the standard Push API, which is available in Loki from its initial release. Since OTLP is a standardized format not specifically designed for Loki, it requires additional configuration on Loki's side to map OpenTelemetry's data format to Loki's data model. OTLP lacks concepts such as *stream labels* or *structured metadata*. Instead, OTLP provides metadata about log entries as *attributes*, grouped into three categories:
+
+* Resource
+* Scope
+* Log
+
+This allows metadata to be set for multiple entries simultaneously or individually as needed.
+
+include::modules/log6x-configuring-lokistack-otlp-data-ingestion.adoc[leveloffset=+1]
+
+== Attribute mapping
+
+Loki splits the configuration for mapping OTLP attribuites to stream labels and structured metadata into two places:
+
+*  `default_resource_attributes_as_index_labels` in the link:https://grafana.com/docs/loki/latest/configure/#distributor[`distributor` configuration]
+* `otlp_config` in the link:https://grafana.com/docs/loki/latest/configure/#limits_config[`limits_config`]
+
+By default, `default_resource_attributes_as_index_labels` maps certain resource attributes to stream labels within Loki. 
+
+Since this distributor configuration only applies to resource-level attributes and generates stream labels as output, the `otlp_config` setting allows mapping resource, scope, or log-level attributes to structured metadata.
+
+[NOTE]
+====
+Attributes that remain unmapped to either stream labels or structured metadata do not store in Loki.
+====
+
+The Loki Operator handles attribute configuration differently than Loki itself. Notably, the Loki Operator does not distinguish between `distributor` and `limits` configurations. Instead, the Loki Operator consolidates all attributes in the `limits` configuration. For more infomation on Operator-specific configuration options, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#custom-attribute-mapping[Custom attribute mapping].
+
+=== Loki Operator defaults
+
+When using the Loki Operator in `openshift-logging` mode, default attribute mappings follow OpenShift configurations. For more information on configuring custom attributes within `openshift-logging` mode, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#customizing-openshift-defaults[Customizing OpenShift defaults].
```

Comment:
This feels like duplication with the entry paragraph in the `Custom attribute mapping`. I suggest to choose one of the two to refer to the `Customizing OpenShift defaults`. Fact is that customizing attribute mapping is the logical next step to deviate from the defaults.

Reply:
To avoid duplication, removing the **Loki Operator defaults**  section seems better option to me.